src/indent.cpp

```diff
@@ -4717,36 +4717,17 @@ void indent_preproc()
                               ? pc->pp_level - pp_level_sub : 0;
 
       // Adjust the indent of the '#'
-      log_rule_B("pp_indent");
-
       if (options::pp_indent() & IARF_ADD)
       {
-         log_rule_B("pp_indent_count");
+         log_rule_B("pp_indent ADD");
```

Comment:
Please use the exact text from the option name

Reply:
@gmaurel 
Well, IMO it is correct to use 'pp_indent' rather than 'pp_indent_count'. 
''pp_indent' determines whether we add or remove indentation here. 'pp_indent_count' only gives the amount of indent when 'pp_indent' is set to add/force the indentation.
You can see the genetic `log_rule_B("pp_indent");` has been removed from line 4720 and instead added to the body of the if/else-if only when executed. If we don't execute any of the if/else-if bodies, we shouldn't print the log rule at all. I also add ADD or REMOVE to specify the actual path of the if/else-if block